samples/net/wpan_serial/src/main.c

```diff
@@ -56,12 +56,13 @@ static uint8_t slip_buf[1 + 2 * CONFIG_NET_BUF_DATA_SIZE];
 
 /* ieee802.15.4 device */
 static struct ieee802154_radio_api *radio_api;
-static const struct device *const ieee802154_dev =
+static const struct device *ieee802154_dev =
 	DEVICE_DT_GET(DT_CHOSEN(zephyr_ieee802154));
```

Comment:
Why is the `const` removed here (same applies to `samples/net/wpanusb/src/wpanusb.c` in the same commit)?

Reply:
already fixed

Reply:
Not really. See cc5ce04dde4cd17fea71cdc78b506e3a9e9a37b0. The change is then reverted in https://github.com/zephyrproject-rtos/zephyr/pull/49167/commits/2548ce925c341b5be34a6825d67360306a5f78e8#diff-7a5b555058d53505817a540a1c8e9e5817e12132c5bd316c890552dfbc2b5170L59-R59.

Reply:
I see, that looks like a rebase artifact will fix it. thanks :-)